Assets/MRTK/Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubConfigurationProfile.asset

```diff
@@ -33,15 +33,15 @@ MonoBehaviour:
   teleportSystemType:
     reference: Microsoft.MixedReality.Toolkit.Teleport.MixedRealityTeleportSystem,
       Microsoft.MixedReality.Toolkit.Services.TeleportSystem
-  enableSpatialAwarenessSystem: 0
+  enableSpatialAwarenessSystem: 1
   spatialAwarenessSystemType:
     reference: Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessSystem,
       Microsoft.MixedReality.Toolkit.Services.SpatialAwarenessSystem
-  spatialAwarenessSystemProfile: {fileID: 11400000, guid: 97da727944a3d7b4caf42d2273271a24,
+  spatialAwarenessSystemProfile: {fileID: 11400000, guid: 7d9fb00aadb946e44b414aad73ec0507,
     type: 2}
   diagnosticsSystemProfile: {fileID: 11400000, guid: 478436bd1083882479a52d067e98e537,
     type: 2}
-  enableDiagnosticsSystem: 0
+  enableDiagnosticsSystem: 1
```

Comment:
Do we want the visual profiler on for the examples hub? Looks like it was previously off:
```suggestion
  enableDiagnosticsSystem: 0
```

Reply:
Originally I turned it off but we have 'Profiles' button in the existing Toggle Features Panel in each scene. Also it is highlighted by default so if we don't activate this, Profiler button won't work and seems broken.